xds/src/main/java/io/grpc/xds/internal/sds/SdsProtocolNegotiators.java

```diff
@@ -281,6 +281,9 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         SslContextProviderSupplier sslContextProviderSupplier = InternalProtocolNegotiationEvent
                 .getAttributes(pne).get(ATTR_SERVER_SSL_CONTEXT_PROVIDER_SUPPLIER);
         if (sslContextProviderSupplier == null) {
+          logger.log(Level.INFO, "No sslContextProviderSupplier found in filterChainMatch "
```

Comment:
This may be normal. INFO is logged by default, and we don't want to log by default when nothing is wrong. Use FINE level?

Ugh. Looks like the "Using fallback" log also needs fixing.

Reply:
And when fallback exists I think it will be nice to have a single message that includes "no ... found and using fallback for ...". 

Also do we need to print out variable names like `sslContextProviderSupplier` or can we just say no filter-matched for connection ..."?

Reply:
It's not an error, so no log. Everything is working as expected. We also have to be careful on server-side not to log too frequently; if every connection fails the same way it can produce a lot of logs very quickly.